### PR TITLE
answers for lesson 1

### PR DESCRIPTION
**Pull request by cecerodrgz**

### CODE REVIEW

File: 01-node-tutorial/answers/intro.js
Comment (PeterMace) [inline 01-node-tutorial/answers/intro.js:1]: Please use the same filename as in the directions. On real projects, naming is key to organization and there can be many files in use.

File: 01-node-tutorial/answers/modules.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+//3a
+const names = require('../04-names')
+console.log(names)
+
+//3b
+const sayHi = require('../05-utils')
```
Comment (PeterMace) [inline 01-node-tutorial/answers/modules.js]: Please also add the files being referenced

File: 01-node-tutorial/answers/path.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const stringToJoin = ['content', 'subfolder', 'text.tx']
```
Comment (PeterMace) [inline 01-node-tutorial/answers/path.js:1]: Please use the OS module, more info here.

https://nodejs.org/api/os.html

File: 01-node-tutorial/09-path-module.js [hunk 3]
```diff
@@ -9,4 +9,4 @@ const base = path.basename(filePath)
 console.log(base)
 
 const absolute = path.resolve(__dirname, 'content', 'subfolder', 'test.txt')
```
Comment (PeterMace) [inline 01-node-tutorial/09-path-module.js:11]: pleas use path.join

File: 01-node-tutorial/10-fs-sync.js [hunk 4]
```diff
@@ -8,5 +8,6 @@ writeFileSync(
   `Here is the result : ${first}, ${second}`,
```
Comment (PeterMace) [inline 01-node-tutorial/10-fs-sync.js:8]: I think this will still only be two lines.

File: 01-node-tutorial/answers/prompter.js [hunk 5]
```diff
@@ -20,18 +20,24 @@ const getBody = (req, callback) => {
   });
 };
```
Comment (PeterMace) [inline 01-node-tutorial/answers/prompter.js:21]: This doesn't seem to compile for me.

Comment (cecerodrgz) [reply 01-node-tutorial/answers/prompter.js:21]: I just tried it and it works just fine, no idea how to fix it, since it works on my end. Any ideas?

Comment (PeterMace) [reply 01-node-tutorial/answers/prompter.js:21]: It is now working for me, sorry that may have been due to the way I was trying to run it. All good!

**Pull request merged**